packages/pinia-orm/src/composables/useRepo.ts

```diff
@@ -5,7 +5,7 @@ import { Database } from '../database/Database'
 import type { Constructor } from '../types'
 import { registerPlugins } from '../store/Plugins'
 
-export function useRepo<R extends Repository> (
+export function useRepo<R extends Repository<any>> (
```

Comment:
I don't get why it stopped working in the first place... Is it somehow related to the attribute autocompletion?